### DISCUSSION TITLE
Need help with jinja2 field for accepted entries count

### DISCUSSION THREAD

**Discussion by spice42** [Flexget/Flexget#3615] [Q&A] [answered]
.

**Comment by gazpachoking** [comment, marked as answer]
If you are using the `task` section of the notify plugin the template will be rendered once per task, rather than once per entry. You can then use the special variable `task` in your template. To get the number of accepted entries in your template it would be `{{ task.accepted|length }}`. Looks like our documentation is a bit lacking on this at the moment.

**Comment by spice42** [comment]
.
